docs/config-definition.md

```diff
@@ -1,9 +1,9 @@
 # Ark Config definition
 
-* [Overview][10]
```

Comment:
I already knew I could just append the new link but thought it'd be nicer if the order of the links and the numbers would match. I'll change it, no problem.